e2etest/pkg/executor/executor.go

```diff
@@ -68,3 +68,25 @@ func (p *podExecutor) Exec(cmd string, args ...string) (string, error) {
 	out, err := exec.Command(Kubectl, fullargs...).CombinedOutput()
 	return string(out), err
 }
+
+// add ephemeral container to deal with distroless image
+type podDebugExecutor struct {
+	namespace string
+	name      string
```

Comment:
wdyt about addibg an image for the container here (instead of defaulting to busybox)? this will make the call for `ForPodDebug` more resemblant to the usual `kubectl debug` where the caller specifies the image.

Reply:
added the randomized string for debugger.

Reply:
image name is configurable now.